architecture-decision-records/README.md

```diff
@@ -1,30 +1,74 @@
-# Architecture decision record (ADR)
+# Architecture decision records (ADRs)
 
 This folder contains Architecture decision records (ADRs) for the Operations Platform.  
 The purpose is to provide transparency and enable a broader team to contribute to various aspects.
 
-## Conventions
+## Illustrations
 
-### Template
+Use [mermaid](https://www.mermaidchart.com/app/dashboard) within the markdown files for technical illustrations instead
+of external assets.
 
-Use the [ADR template](0_template.md).
+## Development
 
-### Illustrations
+Pre-requisites:
 
-Use [mermaid](https://www.mermaidchart.com/app/dashboard) within the markdown files for technical illustrations instead of external assets.
+- Node.js [LTS](https://nodejs.org/en/download/)
 
-### Naming
+**If not already done, install `Log4brains`**
 
-The files should be named using this conventions `<component>-ADR-<number>-<name>.md`.  
-The `<number>` consists of 3 digits. Use leading zeros if necessary.  
-The `CloudOperators` component is used for ADRs of general nature.   
+```bash
+npm install -g log4brains
+```
+
+**To create a new ADR interactively, run**:
+
+```bash
+log4brains adr new
+```
+
+- When prompted for `Title of the solved problem and its solution?` Provide a title in the
+  format `<number> <component> <name>`
+
+- This will generate a new ADR file in the `architecture-decision-records` folder with a filename in the format
+  `<yyyymmdd>-<provided title>.md`
+
+### Rename generated ADR filename
+
+`log4brains` will generate the file with current date as prefix followed by the title provided in the
+command `log4brains adr new` separated by `-`
+
+e.g. `20240101-<provided-title>.md`
+
+- Please rename the generated filename to `<number>-<component>-<provided-title>.md`.
+- The `<number>` should consists of three digits. Use leading zeros if necessary.
+- Component can be one of `greenhouse` | `cloudoperators`.
+
+> The `cloudoperators` component is used for ADRs of general nature.
 
 Example:
-* `Greenhouse-ADR-001-logical_authorization_concept_for_plugins.md`
-* `CloudOperators-ADR-<number>-<name>.md`
 
-### Contributing
+- `001-greenhouse-logical-authorization-concept-for-plugins.md`
+- `002-cloudoperators-<name>.md`
+
+**Alternatively** you can use the `Makefile` command to create a new ADR without any manual renaming
+
+example:
+
+```bash
+make init TITLE=new-adr-title
+```
+This will automatically append the next index and `greenhouse` prefix for the ADR and create a new ADR file with the provided title.
+
+> NOTE: You can also override the prefix by passing PREFIX=<greenhouse | cloudoperators | ...>
+
```

Comment:
## Start the webserver

`log4brains` provides the possibility to serve a web ui showcasing all ADRs in this directory.

Start the web ui from repository root directory as such:
```bash
log4brains preview
  ```
This will bring up an UI on [localhost:4004](http://localhost:4004/)

Reply:
I removed this due to the fact that tables in log4brains and images don't work well. Instead focusing on standardizing ADR(s) template.

These markdowns can be visualized way better in `Hugo` if we decide to merge the docs and adrs in one location or separately in something like `MkDocs` in the future.